src/webui/www/private/scripts/client.js

```diff
@@ -781,8 +781,26 @@ window.addEvent('load', function() {
         addMainWindowTabsEventListener();
         addSearchPanel();
     }
+
+    registerMagnetHandler();
+    handleMagnetParam();
 });
 
+function registerMagnetHandler() {
+    const urlObject = new URL(location.href);
+    urlObject.searchParams.set('magnet', '');
+    const templateUrl = urlObject.href.replace('magnet=', 'magnet=%s');
```

Comment:
what about other parameters in `urlObject.searchParams`? should you ditch them too?

Reply:
This is difficult to answer. I was torn on whether or not to strip any other query params out. In my case, there are no additional query parameters, but I figured it was better to leave any other query parameters in, just in case someone had some case that relies on them. I'm fine with stripping them out if you think that is preferable.

If we leave them in, it means that any query parameters present when the user authorizes the handler will also be passed along with the magnet param when opening magnet links. If we strip them out then only the magnet param will be passed.